docs/fundamentals/documentarray/interaction-cloud.md

```diff
@@ -0,0 +1,41 @@
+(interaction-cloud)=
+# Interaction with Jina AI Cloud
+
+```{important}
```

Comment:
@samsja I believe this `note` is not needed right? they come when u install docarray with `pip install docarray`?

Reply:
yes not needed